### ISSUE TITLE
How to access collection with "-" in name

### ISSUE THREAD

**Issue by AlexBlokh** [microsoft/vscode-cosmosdb#1193]
How do I access collection with "-" in name?

<img width="458" alt="Screen Shot 2019-09-09 at 16 56 57" src="https://user-images.githubusercontent.com/4045375/64537076-d6f15600-d322-11e9-9057-eb9003fa8a92.png">

Repro steps:

1. db.{collection-name}.find() or db.getCollection('collection-name').find()

Action: cosmosDB.executeMongoCommand
Error type: Error
Error Message: collection name must be a String

Version: 0.11.0
OS: darwin
Product: Visual Studio Code
Product Version: 1.38.0
Language: en

<summary>Call Stack</summary>
```
module.exports.t.checkCollectionName extension.bundle.js:5:53946
new C extension.bundle.js:109:8503
N.module.exports.N.collection extension.bundle.js:129:33155
MongoFindResultEditor.<anonymous> extension.bundle.js:622:1082492
s extension.bundle.js:622:1081907
process._tickCallback next_tick.js:68:7
```

**Comment by PrashanthCorp** [comment]
Use `db.getcollection(<hyphened-name>)`

**Comment by PrashanthCorp** [comment]
#981  is related.

**Comment by vilic** [comment]
@PrashanthCorp this makes the extension unusable for us, really hope it gets fixed soon.

**Comment by nturinski** [comment]
Hi @vilic and @AlexBlokh 

While this issue isn't resolved, with the release of v0.12.0, there is now a workaround for this.  If you need to access a collection with a hyphen ("-") in the name, please use `getCollection(col-name)`.

Hope that this helps while we solve the actual problem. Have a great weekend!
